packages/file-upload/src/lib/file-upload.stories.mdx

```diff
@@ -20,84 +20,109 @@ yarn add @novatics/file-upload
   <Story name="Basic Usage">
     {(args) => {
       return (
-        <div style={{ display: 'flex', flexDirection: 'row' }}>
-          <FileUpload
-            fromUrlProps={{ accept: 'image/jpeg', acceptLabels: '.JPEG' }}
+        <Stack width={500} spacing={2}>
+          <DragAndDrop
+            variant="inline"
+            multiple
+            acceptedLabels="JPG, PNG and GIF"
+          />
+          <Divider>or</Divider>
+          <FromUrl
+            accept="image/jpeg"
+            acceptLabels=".JPEG"
+            variant="combined"
           />
-        </div>
+          <br />
+          <ListItem fileName="Test.png" loadingPercentage={25} />
+        </Stack>
       );
     }}
   </Story>
 </Canvas>
 
-```tsx
-import FileUpload from '@novatics/file-upload';
-
-<FileUpload />;
-```
-
 ### From URL
 
+The FromUrl component receives an URL and a callback function to be called when the user clicks on the button to upload the file. It has two variants: `single` and `combined`.
+
 <Canvas>
   <Story name="From URL">
     {(args) => {
       return (
-        <div style={{ display: 'flex', flexDirection: 'row' }}>
-          <FromUrl />
-        </div>
+        <Stack spacing={2}>
+          <FromUrl variant="single" />
+          <FromUrl acceptLabels=".CSV, .XLSX" variant="combined" />
+        </Stack>
       );
     }}
   </Story>
 </Canvas>
 
 ### Drag and Drop
 
+The `DragAndDrop` component can be used in two variants: `inline` and `fullscreen`. Both draggin and dropping files or clicking on the button to select them are supported.
+
 <Canvas>
   <Story name="Drag and Drop">
     {(args) => {
       return (
-        <div>
+        <Stack spacing={2}>
           <DragAndDrop
             variant="fullscreen"
             multiple
             acceptedLabels="JPG, PNG and GIF"
           />
-          <br />
           <DragAndDrop variant="inline" acceptedLabels="JPG, PNG and GIF" />
-        </div>
+        </Stack>
       );
     }}
   </Story>
 </Canvas>
 
-```tsx
-import { FromUrl, FromUrlProps } from '@novatics/file-upload';
+### List Item
 
-<FromUrl />;
-```
+A list item can be used to show the status of a file upload. If it is loading, succeeded or failed.
+
+<Canvas>
+  <Story name="List Item">
+    {(args) => {
+      return (
+        <Stack width="50rem">
+          <ListItem fileName="Test.jpg" loadingPercentage={25} />
+          <br />
+          <ListItem
+            fileName="Test.csv"
+            succeeded
+            onSuccessClick={() => {}}
+            deleteAfterSuccess
+          />
+          <br />
+          <ListItem fileName="Test.txt" error="Tente novamente" />
+        </Stack>
+      );
+    }}
+  </Story>
+</Canvas>
 
 ### Playground
 
 <Canvas>
   <Story name="Playground" argTypes={{}}>
     {(args) => {
-      return (
-        <div style={{ display: 'flex', flexDirection: 'row' }}>
-          <FileUpload {...args} />
-        </div>
-      );
+      return <FileUpload {...args} />;
```

Comment:
I'm not really happy with this playground, there are too many props on each component, maybe it would be better to have a playground for each component?

Reply:
we could move to csf format 

https://storybook.js.org/docs/react/writing-stories/stories-for-multiple-components